src/client/components/home/Search.js

```diff
@@ -0,0 +1,67 @@
+import InputBase from '@material-ui/core/InputBase';
+import Divider from '@material-ui/core/Divider';
+import IconButton from '@material-ui/core/IconButton';
+import SearchIcon from '@material-ui/icons/Search';
+import Paper from '@material-ui/core/Paper';
+import React from 'react';
+import { makeStyles } from '@material-ui/core';
+import PropTypes from 'prop-types';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    padding: theme.spacing(0.25, 0.5),
+    margin: '100px 0px',
+    display: 'flex',
+    alignItems: 'center',
+    width: '100%',
+  },
+  iconButton: {
+    padding: 10,
+  },
+  divider: {
+    height: 28,
+    margin: 4,
+  },
+}));
+
+function Search({ handleSearch, isLoading }) {
+  const classes = useStyles();
+  return (
+    <Paper component="div" className={classes.root}>
+      <InputBase
+        id="search"
+        disabled={isLoading}
+        style={{ margin: 8 }}
```

Comment:
Factor out in `useStyles` . Plus there's a `margin` property, maybe you just need to set it to `dense`.